config/application.rb

```diff
@@ -16,5 +16,18 @@ module HipsterBytes
   class Application < Rails::Application
     config.active_record.raise_in_transactional_callbacks = true
     config.exceptions_app = self.routes
+
+    config.action_mailer.perform_deliveries = true
+    config.action_mailer.delivery_method = :smtp
+
+    ActionMailer::Base.smtp_settings = {
+      :user_name => ENV["SENDGRID_USERNAME"],
+      :password => ENV["SENDGRID_PASSWORD"],
+      :domain => 'hipsterbytes.herokuapp.com',
+      :address => 'smtp.sendgrid.net',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.